### ISSUE TITLE
[SUGGESTION] Brick Corners

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Skywanderers#1030]
As great as the current sizeable bricks are, it seems to me that we are held back by only three things: brick tetras, pentas, and heptas. We have these in block form but it really limits our ability to build and detail when we don't have these crucial shapes in brick form.

Now I don't know how many times you have been suggested this nor how difficult this could be to implement, but i think that if these were to be added to the game it would make it easier to make better looking builds without resorting to clipping bricks (which is still possible in certain situations).

**Comment by tsunamayo** [comment]
Yes 100% true, I wanted to add them but never took the time to do it. Will do soon!
Thanks
